packages/client/utils/constants.ts

```diff
@@ -158,3 +160,27 @@ export const MAX_SPOTLIGHT_COLUMNS = 3
 export const SPOTLIGHT_TOP_SECTION_HEIGHT = 236
 
 export const PARABOL_AI_USER_ID = 'parabolAIUser'
+
+export const TeamBenefits = [
+  'Unlimited teams',
+  'Premium templates',
+  'Custom templates',
+  'Unlimited meeting history',
+  'Priority customer support'
+]
+
+export const readableReasonsToDowngrade: ReadableReasonToDowngradeEnum[] = [
+  'Parabol is too expensive',
+  'Budget changes',
+  'Missing key features',
+  `Not using Parabol's paid features`,
+  'Moving to another tool (please specify)'
+]
+
+export const reasonsToDowngradeLookup = {
+  'Parabol is too expensive': 'tooExpensive',
+  'Budget changes': 'budgetChanges',
+  'Missing key features': 'missingKeyFeatures',
+  "Not using Parabol's paid features": 'notUsingPaidFeatures',
+  'Moving to another tool (please specify)': 'anotherTool'
+} as Record<ReadableReasonToDowngradeEnum, ReasonToDowngradeEnum>
```

Comment:
We don't get the types with Object.keys unless we cast them, so I thought this would be safer

Reply:
+1 If you're going for type safety, then you should not cast with `as` at the end. You can try this and add an invalid key and it will still work. If you instead declare the type of the variable, then this would be detected
```ts

export const reasonsToDowngradeLookup: Record<ReadableReasonToDowngradeEnum, ReasonToDowngradeEnum> = {
  'Parabol is too expensive': 'tooExpensive',
  'Budget changes': 'budgetChanges',
  'Missing key features': 'missingKeyFeatures',
  "Not using Parabol's paid features": 'notUsingPaidFeatures',
  'Moving to another tool (please specify)': 'anotherTool'
}
```